hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/SerialNumberManager.java

```diff
@@ -115,6 +119,36 @@ public static StringTable getStringTable() {
     return map;
   }
 
+  // returns serial snapshot of current values for a save.
+  public static StringTable getSerialStringTable() {
```

Comment:
Please add test cases.

Reply:
Okay, I 'll add this test case as soon as possible.